setup.py

```diff
@@ -1,13 +1,19 @@
+from __future__ import absolute_import
 from setuptools import setup, find_packages
 from oauth_provider import __version__ as version
 import os
 
+
 def strip_comments(l):
     return l.split('#', 1)[0].strip()
 
+
 def reqs(*f):
-    return list(filter(None, [strip_comments(l) for l in open(
-        os.path.join(os.getcwd(), *f)).readlines()]))
+    return list([_f for _f in [strip_comments(l) for l in open(
+        os.path.join(os.getcwd(), *f)).readlines()] if _f])
+
+
+os.system('pip install git+https://github.com/joestump/python-oauth2.git@master#egg=oauth2')
```

Comment:
@feanl find out this solution. not idea but working fine.

Reply:
We shouldn't need this if we update the test requirements correctly.